docs/examples/config.rb

```diff
@@ -0,0 +1,78 @@
+# config/initializers/redis_memo.rb
+RedisMemo.configure do |config|
+  # Static references: Those references are fixed when booting the app
+  config.expires_in = 3.hours
+
+  # On non-heroku environments, clear the cache when starting a new Rails
+  # server process
+  config.global_cache_key_version = ENV['HEROKU_SLUG_COMMIT']
```

Comment:
Hmm it feels blurred to me when reading non heroku environments with the `HEROKU_SLUG_COMMIT` Maybe also add a comment here for what to do in Heroku environments, and the example we provide is for Heroku env?.